README.md

```diff
@@ -270,10 +276,10 @@ To manage this, it is RECOMMENDED that varsig types include a nested encoding mu
 And another showing data signed with [EIP-191]:
 
 ``` abnf
-; secp256k1  EIP-191 message-length  sig-bytes
-;     |         |           |           |
-;     v         v           v           V
-%x34 %xE7 %x1B %xE191 %x45 %x0100 64(OCTET)
+; secp256k1  EIP-191      sig-bytes
+;     |         |            |
+;     v         v            V
+%x34 %xE7 %x1B %xE191 %x45 64(OCTET)
```

Comment:
```suggestion
%x34 %xE7 %x1B %xE191 64(OCTET)
```

Technically `%x45` is part of the message that is being signed, so doesn't need to be included here.

Reply:
Ah indeed, this was leftocer from when we encoded EIP-191 versions. Thanks!